src/Template/Layout/js/app/components/property-view/property-view.js

```diff
@@ -131,5 +149,28 @@ export default {
                 this.$refs.relation.loadRelatedObjects();
             }
         },
+        async loadInfoUsers() {
+            const creatorId = this.object?.meta?.created_by;
+            const modifierId = this.object?.meta?.modified_by;
+            let usersId = [creatorId, modifierId];
+            const userRes = await fetch(`${API_URL}api/objects?filter[id]=${usersId.join(',')}`, API_OPTIONS);
+            const userJson = await userRes.json();
+            const users = userJson.data;
+
+            users.map((user) => {
+                const href = `${BEDITA.base}/${user.type}/view/${user.id}`;
+                const userInfo = (user.attributes.name  != undefined || user.attributes.surname != undefined) 
+                        ? user.attributes.name + ' ' + user.attributes.surname
+                        : user.attributes.uname;
+
+                if(user.id == creatorId) {
```

Comment:
```suggestion
                if (user.id == creatorId) {
```
chore fix: space

Reply:
`user.id` is a String and `creatorId` is a Integer. Do you suggest to cast `user.id` in a integer and after apply this operator?

Reply:
you can use `==`, fine. maybe an inline comment, to explain why, as `// user.id is string, creatorId is int